doc/user/content/sql/patterns/ttl.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Time-To-Live"
+description: "Use the time-to-live pattern and assign an expiration time for a particular row."
+menu:
+  main:
+    parent: 'sql-patterns'
+---
+
+The **time to live (TTL)** pattern helps to remove rows in views, queries, tails, or sinks using expiration times, keeping rows until they are no longer useful, and enabling new use cases.
+
+The following examples are possible TTL use cases:
+
+- Trigger schedule tasks, like emails or messages
+- Store temporal offers for an e-commerce
+- Maintain time-sensitive security blockers
+- Saving-up memory
+
+Before continuing, make sure to understand how [mz_now()](/sql/functions/now_and_mz_now/) and [temporal filters](/sql/patterns/temporal-filters/) work.
+
+## Pattern
+
+The pattern uses a temporal filter over a row's creation timestamp plus a TTL. The sum represents the row's **expiration time**. After reaching the expiration time, the query will drop the row.
+
+Pattern example:
+```sql
+  CREATE VIEW TTL_VIEW
```

Comment:
I think you are still missing an AS, here, but otherwise LGTM!

Reply:
Correct!